runtime/bcverify/bcverify.c

```diff
@@ -979,11 +978,11 @@ mergeStacks (J9BytecodeVerificationData * verifyData, UDATA target)
 								*targetStackPtr = *sourceStackPtr;
 								rewalk = TRUE;
 							} else {
-
 								/* Use local mapper to check merge necessity in locals */
 								if ((verifyData->verificationFlags & J9_VERIFY_OPTIMIZE) && (maxIndex <= 32)) {
```

Comment:
`verifyData->verificationFlags & J9_VERIFY_OPTIMIZE` is the kind of thing I've seen cause compilation problems in the past on P. If these types of expressions are modified to use J9_ARE_ALL_BITS_SET() or J9_ARE_ANY_BITS_SET() which create proper boolean expressions, it might help.

Reply:
The code has been there for quite a long while prior to OpenJ9. If there is any change involved, extensive tests must be carried out on all platforms to ensure there is no issue introduced if anything.

Reply:
I can go either way on this one - it has nothing to do with what's being changed in this PR, but I don't believe it would introduce any errors, even on our more deficient compilers.

Reply:
I just double-checked `J9_VERIFY_OPTIMIZE` in the code. Given it is used by default, technically it should be fine if the code here is replaced with `J9_ARE_ALL_BITS_SET()` or `J9_ARE_ANY_BITS_SET()`.

Reply:
Note I'm not saying this particular check is causing #11086, it's just the first one like this I saw. If we were to make this type of change, all expressions similar to this one should be changed. i.e. a bit test that doesn't compare with anything.

Reply:
I'm also not suggesting this PR needs to be updated, it was just a convenient place to point it out.

Reply:
I find it hard to believe the implicit 0 check is causing any problem, ever, given that C code uses this extensively (including thousands of occurrences in our own code).

Reply:
There's also some odd code pulling things into locals and then continuing to deref the original pointers.  I'd like to see it cleaned up as well but it's less mechanical.  Look at
```
	UDATA sourceItem = *sourceStackPtr;
	UDATA targetItem = *targetStackPtr;
```